Website/src/rendering/src/components/FeaturedSpeakers.tsx

```diff
@@ -1,71 +1,65 @@
-        <Link href="/speakers/mary-asada" passHref>
-          <div className="rounded overflow-hidden">
-            <Image src={speaker1} alt="Speaker" width={265} height={265} />
-            <div className="px-6 py-4">
-              <p className="text-gray-700 text-base text-center">Mary Asada</p>
-            </div>
-          </div>
-        </Link>
+type ContentListProps = ComponentProps & {
+  fields: {
+    Title: Field<string>;
+    Subtitle: Field<string>;
+    Speakers: Speaker[];
+  };
+};
 
-        <Link href="/speakers/jacobgonzalez" passHref>
-          <div className="rounded overflow-hidden">
-            <Image src={speaker2} alt="Speaker" width={265} height={265} />
-            <div className="px-6 py-4">
-              <p className="text-gray-700 text-base text-center">Jacob Gonzalez</p>
-            </div>
-          </div>
-        </Link>
-
-        <Link href="/speakers/edjones" passHref>
-          <div className="rounded overflow-hidden">
-            <Image src={speaker3} alt="Speaker" width={265} height={265} />
-            <div className="px-6 py-4">
-              <p className="text-gray-700 text-base text-center">Ed Jones</p>
-            </div>
-          </div>
-        </Link>
-
-        <Link href="/speakers/sophiataylor" passHref>
-          <div className="rounded overflow-hidden">
-            <Image src={speaker4} alt="Speaker" width={265} height={265} />
-            <div className="px-6 py-4">
-              <p className="text-gray-700 text-base text-center">Sophia Taylor</p>
-            </div>
-          </div>
-        </Link>
-
-        <Link href="/speakers/lixiuying" passHref>
-          <div className="rounded overflow-hidden">
-            <Image src={speaker5} alt="Speaker" width={265} height={265} />
-            <div className="px-6 py-4">
-              <p className="text-gray-700 text-base text-center">Li Xiu Ying</p>
-            </div>
-          </div>
-        </Link>
+const FeaturedSpeakers = (props: ContentListProps): JSX.Element => {
+  return (
+    <section className="">
+      <div className="max-w-screen-2xl mx-auto box-border overflow-hidden bg-white">
+        <Text
+          tag="h1"
+          className="text-center uppercase text-blue pt-10 text-3xl md:text-4xl font-semibold"
+          field={props.fields?.Title}
+        />
+        <Text tag="p" className="text-center" field={props.fields?.Subtitle} />
+        <div className="p-10 grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5 gap-5">
+          {props.fields.Speakers &&
+            props.fields.Speakers.map((speaker, index) => {
+              return (
+                <Link
+                  key={index}
+                  href={'/speakers/' + speaker.fields.Name.value.replace(/ /g, '')}
+                  passHref
+                >
+                  <div className="rounded overflow-hidden cursor-pointer mx-auto">
```

Comment:
`cursor-pointer` should be removed and replaced by a proper `<a>` element which is missing at the moment in the HTML output.